lib/identity_cache/configuration_dsl.rb

```diff
@@ -217,6 +217,8 @@ def build_id_embedded_has_many_cache(association, options) #:nodoc:
         options[:records_variable_name]   = "cached_#{association}"
         options[:prepopulate_method_name] = "prepopulate_fetched_#{association}"
 
+        inverse_name = options[:association_reflection].inverse_of.try(:name)
```

Comment:
Prefer `try!` over `try`, since the latter also does a respond_to? check that can hide bugs.